clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -8446,28 +8446,46 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
     TranslatorArgs.push_back("-spirv-allow-extra-diexpressions");
     TranslatorArgs.push_back("-spirv-allow-unknown-intrinsics=llvm.genx.");
 
-    // Disable SPV_INTEL_usm_storage_classes by default since it adds new
-    // storage classes that represent global_device and global_host address
-    // spaces, which are not supported for all targets. With the extension
-    // disable the storage classes will be lowered to CrossWorkgroup storage
-    // class that is mapped to just global address space. The extension is
-    // supposed to be enabled only for FPGA hardware.
-    std::string ExtArg("-spirv-ext=+all,-SPV_INTEL_usm_storage_classes");
+    std::string ExtArg("-spirv-ext=");
+    std::string DefaultExtArg =
+        "+SPV_EXT_shader_atomic_float_add,+SPV_EXT_shader_atomic_float_min_max,"
+        "+SPV_KHR_no_integer_wrap_decoration,+SPV_KHR_float_controls";
```

Comment:
```suggestion
        "+SPV_EXT_shader_atomic_float_add,+SPV_EXT_shader_atomic_float_min_max"
        ",+SPV_KHR_no_integer_wrap_decoration,+SPV_KHR_float_controls";
```

To be consistent with the lines below